src/main/clm_driver.F90

```diff
@@ -325,12 +324,12 @@ subroutine clm_drv(doalb, nextsw_cday, declinp1, declin, rstwr, nlend, rdate, ro
        call get_clump_bounds(nc, bounds_clump)
 
        call t_startf('begcnbal_grc')
-       if (use_cn) then
+       if (use_cn .or. use_fates) then
           ! Initialize gridcell-level balance check
           call bgc_vegetation_inst%InitGridcellBalance(bounds_clump, &
```

Comment:
the argument to the vegetation facade now has the updated soil and patch filters.  However, I held off on changing the local copy of the filters in all the down-stream subroutines to the new naming convention.